notes/crypto/addresses_keys/proof_authorization_keys.md

```diff
@@ -0,0 +1,13 @@
+
+# Proof Authorization Keys
+
+The *proof authorization key* has two components:
+
+* $ak$, the *authorization key*, a point on the `decaf377` curve,
+* $nsk$, the *nullifier private key*, as defined in the [*expanded spending key*](./expanded_spending_keys.md) section,
+
+The authorization key is derived by multiplying the scalar $ask$ (from the expanded spending key) by a fixed generator point [$B$](../primitives/decaf377/test_vectors.md) on the `decaf377` curve (this is analogous to `SpendAuthSig.DerivePublic` in the Zcash specification):
+
+$ak = B * ask$
```

Comment:
```suggestion
$ak = [ask]B$
```

The asterisk doesn't render well in math mode; I think this notation for scalar multiplication is clearer.